pkgs/top-level/all-packages.nix

```diff
@@ -16846,6 +16846,8 @@ in
 
   microsoft_gsl = callPackage ../development/libraries/microsoft_gsl { };
 
+  microsoft-edge-beta = callPackage ../applications/networking/browsers/microsoft-edge { gconf = gnome2.GConf; };
```

Comment:
cc @NixOS/gnome is there a better alternative to using something from gnome2?

Reply:
GConf should not be used at all these days.

Reply:
Both `chromium` and `google-chrome` have this dependency. It seems like upstream dropped GConf in favor of gsettings in 2017, though.

Reply:
Yeah, almost nothing uses it these days. I started cleaning it up (https://github.com/NixOS/nixpkgs/issues/39976) but there are still many instances.